nohup: ignoring input
/Users/moeinfahmide/miniconda3/envs/open-air/lib/python3.12/site-packages/flask_limiter/extension.py:324 UserWarning: Using the in-memory storage for tracking rate limits as no storage was explicitly specified. This is not recommended for production use. See: https://flask-limiter.readthedocs.io#configuring-a-storage-backend for documentation about configuring the storage backend.
/Users/moeinfahmide/miniconda3/envs/open-air/lib/python3.12/site-packages/airflow/api_connexion/schemas/task_schema.py:52 ChangedInMarshmallow4Warning: `Number` field should not be instantiated. Use `Integer`, `Float`, or `Decimal` instead.
/Users/moeinfahmide/miniconda3/envs/open-air/lib/python3.12/site-packages/airflow/api_connexion/schemas/task_schema.py:55 ChangedInMarshmallow4Warning: `Number` field should not be instantiated. Use `Integer`, `Float`, or `Decimal` instead.
/Users/moeinfahmide/miniconda3/envs/open-air/lib/python3.12/site-packages/airflow/api_connexion/schemas/task_schema.py:59 ChangedInMarshmallow4Warning: `Number` field should not be instantiated. Use `Integer`, `Float`, or `Decimal` instead.
/Users/moeinfahmide/miniconda3/envs/open-air/lib/python3.12/site-packages/airflow/providers/fab/auth_manager/fab_auth_manager.py:569 FutureWarning: section/key [webserver/update_fab_perms] has been deprecated, you should use[fab/update_fab_perms] instead. Please update your `conf.get*` call to use the new name
[2025-05-12 15:54:40 +0100] [61032] [INFO] Starting gunicorn 23.0.0

Error: Already running on PID 57387 (or pid file '/Users/moeinfahmide/airflow/airflow-webserver.pid' is stale)

  ____________       _____________
 ____    |__( )_________  __/__  /________      __
____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
 _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
Running the Gunicorn Server with:
Workers: 4 sync
Host: 0.0.0.0:8080
Timeout: 120
Logfiles: - -
Access Logformat: 
=================================================================
[[34m2025-05-12T15:56:38.321+0100[0m] {[34mwebserver_command.py:[0m223} ERROR[0m - No response from gunicorn master within 120 seconds[0m
[[34m2025-05-12T15:56:38.324+0100[0m] {[34mwebserver_command.py:[0m224} ERROR[0m - Shutting down webserver[0m
